src/electionguard-ui/ElectionGuard.UI/ViewModels/BallotUploadViewModel.cs

```diff
@@ -325,14 +327,17 @@ private void UploadMore()
     private readonly CiphertextTallyService _ciphertextTallyService;
     private bool _importing;
     private long _lastDrive = -1;
+    private ILogger<BallotUploadViewModel> _logger;

```

Comment:
The field should not be `ILogger<T>`, even though DI is sending that into it:

```suggestion
    private ILogger _logger;

```